content/participants/ttjc@tarunyadav1/index.md

```diff
@@ -1,15 +1,21 @@
 ---
+
```

Comment:
You have added space after each tag here. Please remove all those spaces.

Reply:
@khusharth  I had remove space and add more projects. Kindly merge my request.Sorry for late reply.
Thank you